src/deployment/linux.md

```diff
@@ -342,10 +316,18 @@ To use the LXD container backend:
 $ snapcraft --use-lxd
 ```
 
-## Publish
+Once the `snapcraft.yaml` file is complete,
+
+## Test the snap
 
 Once the snap is built, you'll have a `<name>.snap` file
-in your root project directory. You can now publish the snap.
+in your root project directory. 
+
+$ sudo snap install ./super-cool-app_0.1.0_amd64.snap --devmode
```

Comment:
It's better to test without devmode.  It's fine to try devmode early in development, but always test without devmode before publishing.